index.js

```diff
@@ -27,6 +28,15 @@ module.exports = function (dir, opts, cb) {
     dat.owner = archive.owner
     dat.resumed = archive.resumed
 
+    // dat.json (TODO: move to module & validate dat.json)
+    var datJsonFile = path.join(dir, 'dat.json')
+    try {
+      dat.meta = JSON.parse(fs.readFileSync(datJsonFile, 'utf8'))
+    } catch (e) {
+      dat.meta = {title: path.basename(dir), description: ''}
+      fs.writeFileSync(datJsonFile, JSON.stringify(dat.meta))
```

Comment:
maybe we could use async methods writeFile and readFile just in case someone is using dat-node in an application since the sync methods will stop the thread